devicetypes/smartthings/zigbee-smoke-sensor.src/zigbee-smoke-sensor.groovy

```diff
@@ -148,6 +179,12 @@ def configure() {
 	Integer minReportTime = 0
 	Integer maxReportTime = 180
 	Integer reportableChange = null
-	return refresh() + zigbee.enrollResponse() + zigbee.configureReporting(zigbee.POWER_CONFIGURATION_CLUSTER, 0x0021, DataType.UINT8, 30, 1200, 0x10) +
-				zigbee.configureReporting(zigbee.IAS_ZONE_CLUSTER, zigbee.ATTRIBUTE_IAS_ZONE_STATUS, DataType.BITMAP16, minReportTime, maxReportTime, reportableChange)
+	return refresh() + 
+			zigbee.enrollResponse() +
+			(isFrientSensor() ? zigbee.configureReporting(zigbee.POWER_CONFIGURATION_CLUSTER, 0x0020, DataType.UINT8, 30, 1200, 0x1) : zigbee.configureReporting(zigbee.POWER_CONFIGURATION_CLUSTER, 0x0021, DataType.UINT8, 30, 1200, 0x10)) +
```

Comment:
One last place to use `BATTERY_VOLTAGE_ATTR` or `BATTERY_PERCENT_ATTR`. :)

Reply:
Or, if you wanted to get super clean, you could do:

```
Integer batteryAttr = isFrientSensor() ? BATTERY_VOLTAGE_ATTR : BATTERY_PERCENT_ATTR
Integer batteryReportChange = isFrientSensor() ? 0x1 : 0x10

...
			zigbee.configureReporting(zigbee.POWER_CONFIGURATION_CLUSTER, batteryAttr, DataType.UINT8, 30, 1200, batteryReportChange) +

...
```

Reply:
Done Donald. Another option to standardize the reporting would be like:
```
Integer batteryReportChange = BATTERY_VOLTAGE_ATTR ? 0x1 : 0x10
```
This would make it report every 100mV for voltage and 16% depending on what attribute is being configured.

Reply:
That is a good point! Though I think you'd want `Integer batteryReportChange = batteryAttr  == BATTERY_VOLTAGE_ATTR ? 0x1 : 0x10`.

I'm honestly not sure why it is 8%. Personally, I would have done 5%. Maybe that is what was originally meant? `10` instead of `0x10`?

Maybe a better choice would be (with the comment as well):

```
Integer batteryReportChange = batteryAttr  == BATTERY_VOLTAGE_ATTR ? 1 : 10 // 100mV or 5%
```

@tpmanley Thoughts?